CHANGELOG.md

```diff
@@ -14,6 +14,18 @@ Releases
 This high level changelog is usually updated when a release is tagged.
 On the master branch there may be changes that are not (yet) described here.
 
+### 1.15.0
+
+* [-] fix issue sending pin codes
+* [+] add command line option to bind to specific interface
+* [+] Use ifferent hints for OTP and 2FA
+* [-] fix coverity defects
+* [-] always define user agent
+* [+] remove password from /proc/#/cmd
+* [+] extend OTP to allow FTM push
+* [+] add support for host checks
+* [-] don't accept route to the vpn gateway
+
 ### 1.14.1
 
 * [-] fix out of bounds array access
```

Comment:
the out of bounds array access was part of the 1.14.1 release, so I don't want to change that line.

Reply:
Ah right. Then it must appear as an individual item.

But then if the rest of the Coverity defects are related to new code added after 1.14.1, we can remove `[-] fix coverity defects`.

Reply:
Shouldn't we still have a changelog item to cover the coverity related commits since 1.14.1: f965ea7f6142a649e8fddd05d3ea12610198ab89,  3b97de302a3ff1735eb9ca18b9910dde85bd58d2, 0dcf211e578e3ff18a2003bacb1c6a9c76b733d6 and bb722111bcddda8666c67fa4c7f852328a7ff00c?

Reply:
If they fix code added **post**-1.14.1, I'd say we do not need to cover in the change log, as the the _master_ branch is a development branch that is supposed to contain bugs.
If they fix code added **pre**-1.14.1, they need to appear in the change log.
I don't know right now which case we're in.

Reply:
Got it. On checking, all those coverity commits are related to code added after 1.14.1, so I removed the line from the changelog.